tests/phpunit/api/v3/ContactTest.php

```diff
@@ -686,7 +686,7 @@ public function testCreateWithCustom(int $version): void {
     $params = $this->_params;
     $params['custom_' . $ids['custom_field_id']] = 'custom string';
     $description = 'This demonstrates setting a custom field through the API.';
-    $result = $this->callAPIAndDocument($this->_entity, 'create', $params, __FUNCTION__, __FILE__, $description);
```

Comment:
Looks like they forgot to copy $subfile here. Clearly an oversight!

(i.e. could remove the $description var in line 688 - I assume your regex/script is also looking for subfile so doesn't see it)